src/lib/apiTypesV2/Entity.cpp

```diff
@@ -38,6 +38,7 @@
 */
 Entity::Entity()
 {
+  typeGiven = false;
```

Comment:
Yes, of course, the title is poorly chosen.

However, a clarification to your comment:

 type: "" (typeGiven = **true**)
absences of type (typeGiven = **false**)